From 57d74f0bb4d86a27698134f42cb92a8596e11cc2 Mon Sep 17 00:00:00 2001
From: Myy Miouyouyou <myy@mioiuyouyou.fr>
Date: Thu, 19 Oct 2017 21:00:12 +0200
Subject: [PATCH 29/33] midgard: mali_kbase: include <linux/sched/task_stack.h>

It seems that most of the old patches are not required anymore.
Except this one :

* sched/headers: Move task-stack related APIs from <linux/sched.h> to
  <linux/sched/task_stack.h>
  f3ac60671954c8d413532627b1be13a76f394c49
  v4.11-rc1

Since this include can be guarded with #if #endif clauses, I don't
know they were not included in the r20p0 version...

Well.

Signed-off-by: Myy Miouyouyou <myy@mioiuyouyou.fr>
---
 drivers/gpu/arm/midgard/mali_kbase.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/arm/midgard/mali_kbase.h b/drivers/gpu/arm/midgard/mali_kbase.h
index 7174ef2ba3d2..db75bc2f1013 100644
--- a/drivers/gpu/arm/midgard/mali_kbase.h
+++ b/drivers/gpu/arm/midgard/mali_kbase.h
@@ -39,6 +39,7 @@
 #include <linux/rwsem.h>
 #include <linux/sched.h>
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(4, 11, 0))
+#include <linux/sched/task_stack.h>
 #include <linux/sched/mm.h>
 #endif
 #include <linux/slab.h>
-- 
2.16.2

